packages/baseclient/config/projectconfig.js

```diff
@@ -1,14 +0,0 @@
-const config = {
-  shogunConfig: {
```

Comment:
you will also need to make use of variables for the `shogunConfig`, or the application will not start with shogun backend anymore

Reply:
:heavy_check_mark: , see 151af25